### ISSUE TITLE
Cleanup team2 and team2-dev

### ISSUE THREAD

**Issue by mdgilene** [rsanchez-wsu/jfiles#235]
A decision needs to be made on what to do with team2 and team2-dev. I have begun looking through the code and my thoughts are below.

## team2-dev
This branch is largely abandoned. It appears that all of the existing team2-dev code exists in master already. So I propose this branch be retired.

## team2
This branch on the other hand appears to contain the majority of the code that team2 was working on.  Their work focused mainly on the edu.wright.cs.jfiles.common package where they developed the beginnings of representing files and a file structure with a way to express files in XML.

fileSystem.dtd appears to be abandoned for the method that exists in the current team2 branch. I do not believe any of the current systems reference fileSystem.dtd so I think we are safe to remove that as well in favor of the system that team2 was working on.

So for the team2 branch I suggest we merge all their changes to master.

Please take a look through the branches and let me know your thoughts as to how we should proceed.

**Comment by LoganRickert** [comment]
Please see issue #225 as well.

**Comment by LoganRickert** [comment]
There is issue #206 as well

**Comment by mdgilene** [comment]
Based on these two issues I am going to go ahead and merge team2 into master and then archive both team2 and team2-dev

**Comment by mdgilene** [comment]
team2-dev has been archived.

master has been merged into team2. Once we are ok with that merge we should be fine to go ahead and merge team2 into master then archive team2.

**Comment by rsanchez-wsu** [comment]
@mdgilene Is your idea that once team2 is merged to master that this issue can be closed?

**Comment by mdgilene** [comment]
Yes that is the idea. I just haven't had the time to do so yet.

**Comment by rsanchez-wsu** [comment]
That sounds good to me.
